packtools/sps/validation/article_and_subarticles.py

```diff
@@ -318,7 +317,7 @@ def validate_article_type(self, article_type_list=None):
         )
 
     def validate_article_type_vs_subject_similarity(
-        self, subjects_list=None, expected_similarity=0
+        self, subjects_list=None, expected_similarity=1
```

Comment:
@Rossi-Luciano adicionar o parâmetro preferencialmente onde ele será usado que é neste método e não na classe.
Senão você está obrigando o usuário da classe sempre passar uma lista desnecessariamente se ele não for usar este método. Claro que agora só tem este método, mas se houver outros?

Reply:
OK, @robertatakenaka